### PR TITLE
Deploying custom models to snowflake model registry quickstart

### PR DESCRIPTION
**Pull request by sfc-gh-mstellwall**
First version for publication

### CODE REVIEW

File: site/sfguides/src/deploying_custom_models_to_snowflake_model_registry/deploying_custom_models_to_snowflake_model_registry.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
+author: Mats Stellwall
+id: deploying_custom_models_to_snowflake_model_registry
+summary: Through this quickstart guide, you will learn how to deploy a custom model to the Snowflake Model Registry.
+<!--- Categories below should be hyphenated, i.e., Getting-Started. Do not leave blank. Visit site for available categories. -->
```
Comment (iamontheinet) [inline site/sfguides/src/deploying_custom_models_to_snowflake_model_registry/deploying_custom_models_to_snowflake_model_registry.md]: @sfc-gh-mstellwall Can you please delete this comment?

**Review by iamontheinet — CHANGES_REQUESTED** [review]

File: site/sfguides/src/deploying_custom_models_to_snowflake_model_registry/deploying_custom_models_to_snowflake_model_registry.md (hunk 1 above)
Comment (sfc-gh-mstellwall) [reply site/sfguides/src/deploying_custom_models_to_snowflake_model_registry/deploying_custom_models_to_snowflake_model_registry.md]: Fixed

File: site/sfguides/src/deploying_custom_models_to_snowflake_model_registry/deploying_custom_models_to_snowflake_model_registry.md [hunk 2]
```diff
@@ -0,0 +1,182 @@
+author: Mats Stellwall
+id: deploying_custom_models_to_snowflake_model_registry
+summary: Through this quickstart guide, you will learn how to deploy a custom model to the Snowflake Model Registry.
+
```
Comment (iamontheinet) [inline site/sfguides/src/deploying_custom_models_to_snowflake_model_registry/deploying_custom_models_to_snowflake_model_registry.md]: @sfc-gh-mstellwall Sorry... can you please delete this blank line. Basically comments or blank lines in this section prevents it from displaying on the site.

**Review by iamontheinet — CHANGES_REQUESTED** [review]

File: site/sfguides/src/deploying_custom_models_to_snowflake_model_registry/deploying_custom_models_to_snowflake_model_registry.md (hunk 2 above)
Comment (sfc-gh-mstellwall) [reply site/sfguides/src/deploying_custom_models_to_snowflake_model_registry/deploying_custom_models_to_snowflake_model_registry.md]: Fixed, sorry.

**Pull request merged**